### PR TITLE
Added CF Turnstile example phpmyadmin

### PR DESCRIPTION
**Pull request by akama-aka**
Added reCaptcha example for Cloudflare Turnstile config example

Cloudflare Developer Page for Turnstile: https://developers.cloudflare.com/turnstile/get-started/

Signed-off-by: SlushUwU <rology@satowa-network.eu>

### CODE REVIEW

File: libraries/classes/Config.php [hunk 1]
```diff
@@ -365,7 +365,7 @@ public function load(?string $source = null): bool
         try {
             /** @psalm-suppress UnresolvableInclude */
             $eval_result = include $this->getSource();
-        } catch (Throwable $exception) {
+        } catch (Throwable) {
```
Comment (williamdes) [inline libraries/classes/Config.php:368]: this is unrelated

**Comment by williamdes** [comment]
can you rebase and push your actual work?
or did you want to open an issue?

**Comment by SlushUwU** [comment]
> 

I re-request it because Idk what happned but something happned on there 😅
